shadowsocks-csharp/Controller/StartupHotkeyReg.cs

```diff
@@ -59,8 +60,12 @@ public static void RegHotkey()
                 }
 
                 foreach (var v in _hotKeyDic)
-                {
-                    HotKeys.Regist(v.Key, v.Value);
+                {
+                    if (!HotKeys.Regist(v.Key, v.Value))
```

Comment:
If you have 4 keys to be registered but fails on the 3rd key. It is different from Hotkey Register Form that user have interactions to change the hotkey definition immediately. Should SS rollback (un-register) the previous 2 keys? OR even popup the Hotkey Register Form with colored notification (green for successful keys, yellow for failed keys) to ask user?